keyboards/handwired/tractyl_manuform/5x6_right/f411/config.h

```diff
@@ -26,6 +26,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define MATRIX_ROW_PINS \
     { B12, B13, B14, B15, A8, A9 }
 
+#define UNUSED_PINS \
+    { B2, C15 }
+// A10,  A11 and A12  don't like to be used?
```

Comment:
Not sure why.  Would be nice to be able to use these, as well.

Reply:
https://discord.com/channels/440868230475677696/440870965728116754/808224716102828042